include/swift/AST/DiagnosticsSema.def

```diff
@@ -6699,5 +6699,31 @@ ERROR(macro_undefined,PointsToFirstBadToken,
       "macro '%0' is undefined; use `-load-plugin-library` to specify dynamic "
       "libraries that contain this macro", (StringRef))
 
+//------------------------------------------------------------------------------
+// MARK: Move Only Errors
+//------------------------------------------------------------------------------
+
+ERROR(moveonly_copyable_type_with_moveonly_subtype, none,
+      "copyable %0 %1 can not contain a move only subtype. Found move only "
```

Comment:
"cannot" is one word, and "subtype" means something else. How about:

`%0 %1 cannot contain a move-only type without also being move-only itself"

and having the "found member with move-only type" be note(s) on the enum cases or properties that have move-only types?

Reply:
Did it